src/test/frontend/logs/logs_controller_test.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import LogsController from 'logs/logs_controller';
+import LogsModule from 'logs/logs_module';
+import {logsFactory} from 'logs/logs_factory';
+
+describe('Logs controller', () => {
+  /**
+   * Logs menu controller.
```

Comment:
You can delete this comment "Logs menu controller". You can read this from its type info :)